### DISCUSSION TITLE
Graceful shutdown of dumphfdl in a shell script?

### DISCUSSION THREAD

**Discussion by jaymot123** [szpajder/dumphfdl#27] [Q&A] [answered]
If I want to run dumphfdl from shell scripts at various times of the day in cron to change the frequencies it's using, is there a way to gracefully shut down the running instance of dumphfdl first rather than doing `pkill -9 -f dumphfdl || true`?

**Comment by szpajder** [comment, marked as answer]
Just omit -9.

**Reply by jaymot123** [reply]
> Just omit -9.

D'oh! I was approaching this from the wrong direction entirely, thinking maybe there's some cleaner way to close the program such as "dumphfdl --exit" that I should be using. Just a plain "pkill dumphfdl" seems to be working. Thanks.
